docs/topics/db/models.txt

```diff
@@ -907,18 +907,13 @@ example::
         name = models.CharField(max_length=100)
         slug = models.TextField()
 
-        def save(
-            self, force_insert=False, force_update=False, using=None, update_fields=None
-        ):
+        def save(self, *args, **kwargs):
```

Comment:
I guess in the ticket description I was also envisioning updating the example just above this, to remove references to the now-deprecated `*args`.

```diff
diff --git a/docs/topics/db/models.txt b/docs/topics/db/models.txt
index cc6c1f5298..4c70ff5662 100644
--- a/docs/topics/db/models.txt
+++ b/docs/topics/db/models.txt
@@ -861,9 +861,9 @@ to happen whenever you save an object. For example (see
         name = models.CharField(max_length=100)
         tagline = models.TextField()
 
-        def save(self, *args, **kwargs):
+        def save(self, **kwargs):
             do_something()
-            super().save(*args, **kwargs)  # Call the "real" save() method.
+            super().save(**kwargs)  # Call the "real" save() method.
             do_something_else()
 
 You can also prevent saving::
@@ -875,23 +875,23 @@ You can also prevent saving::
         name = models.CharField(max_length=100)
         tagline = models.TextField()
 
-        def save(self, *args, **kwargs):
+        def save(self, **kwargs):
             if self.name == "Yoko Ono's blog":
                 return  # Yoko shall never have her own blog!
             else:
-                super().save(*args, **kwargs)  # Call the "real" save() method.
+                super().save(**kwargs)  # Call the "real" save() method.
 
 It's important to remember to call the superclass method -- that's
-that ``super().save(*args, **kwargs)`` business -- to ensure
+that ``super().save(**kwargs)`` business -- to ensure
 that the object still gets saved into the database. If you forget to
 call the superclass method, the default behavior won't happen and the
 database won't get touched.
 
 It's also important that you pass through the arguments that can be
-passed to the model method -- that's what the ``*args, **kwargs`` bit
+passed to the model method -- that's what the ``**kwargs`` bit
 does. Django will, from time to time, extend the capabilities of
-built-in model methods, adding new arguments. If you use ``*args,
-**kwargs`` in your method definitions, you are guaranteed that your
+built-in model methods, adding new arguments. If you use
+``**kwargs`` in your method definitions, you are guaranteed that your
 code will automatically support those arguments when they are added.
 
 If you wish to update a field value in the :meth:`~Model.save` method, you may
```